index.html

```diff
@@ -105,6 +108,68 @@
       </div>
     </header>
     <main :class="`target-${target.length}`">
+      <div class="about-content" x-show="page === '#about'" x-cloak>
+        <p>Spellie is a daily world puzzle geared at young spellers.</p>
+
+        <h1>Features</h1>
+        <ul>
+          <li>Age appropriate puzzles within 3 difficulty levels</li>
+          <li>Easy mode uses shorter words within the Grade 2 vocabulary</li>
+          <li>
+            Hard mode is challenging, but uses words within the Grade 5
+            vocabulary
+          </li>
+          <li>Kid-friendly colours and fonts</li>
```

Comment:
We may want to avoid the word "colour"...

🤔 For valid words, I guess we need an american/british/canadian/australian selection or something...!